### PR TITLE
[DC-2020] Create a DC event. Very TBD.

### PR DESCRIPTION
**Pull request by robbkidd**
DevOpsDays DC 2020 is very TBD in light of COVID-19. Created the skeletonest of an event to prepare a place for more information as it is known.

### CODE REVIEW

**Comment by robbkidd** [comment]
@nathenharvey :eyes: ❓

**Comment by bridgetkromhout** [comment]
Organizer updates need to happen on Slack and email before this is merged.

**Comment by robbkidd** [comment]
@bridgetkromhout An email with organizer names and emails was sent to `info@devopsdays.org`. Is there anything else for me to do to help make the Slack and email updates happen? Or is that a note that there are external task dependencies before merging this PR?

**Comment by bridgetkromhout** [comment]
> Or is that a note that there are external task dependencies before merging this PR?

Correct, letting you know that those tasks also need to happen, so that will all happen together when someone has the time to do them.

File: data/events/2020-washington-dc.yml [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+    twitter: "nathenharvey"
+    bio: ""
+    image: "nathen-harvey.jpg"
+    github: "nathenharvey"
+    facebook: "https://www.facebook.com/nathenharvey"
+    linkedin: "https://www.linkedin.com/in/nathen"
+    website: "http://nathenharvey.com"
+    role: "Co-chairperson"
+
+  - name: "Jonathan Altman"
```
Comment (yvovandoorn) [inline data/events/2020-washington-dc.yml]: @robbkidd Jonathan isn't listed in the email to org. Please advise.

File: data/events/2020-washington-dc.yml [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+  - name: "Jake Vanderdray"
+    employer: "Cvent"
+    twitter: ""
+    bio: ""
+    github: ""
+    facebook: ""
+    website: ""
+    linkedin: ""
+
+  - name: "Dayton Williams"
```
Comment (yvovandoorn) [inline data/events/2020-washington-dc.yml]: Not listed in email to org. Is Dayton to be removed or added?

**Pull request merged**